### DISCUSSION TITLE
difference from langflow？

### DISCUSSION THREAD

**Discussion by linonetwo** [FlowiseAI/Flowise#74] [General]
langflow is a project simillar to this, i saw it months ago.

**Comment by HenryHengZJ** [comment]
yep, langflow is built on Python, and flowise is built on JS/TS. Both of us are working together with the goal to bring people from all backgrounds into LLM space, in the languages they are comfortable to code with

**Reply by jakenuts** [reply]
I for one am thrilled theres a TS option as my native tongue C# is entirely out of this race.

**Reply by linonetwo** [reply]
This is good, since langflow still not supporting langchainTS

**Comment by jingchang0623-crypto** [comment]
Great question! Both Flowise and langflow are visual flow builders for LangChain, but there are some key differences:

**Flowise Advantages:**
- More beginner-friendly UI
- Easier npm/npx deployment (get started in seconds)
- Better API-first architecture
- More affordable commercial licensing for businesses

**langflow Advantages:**
- More mature UI with more components
- Better documentation
- More enterprise-focused features

Both are excellent choices - it depends on your use case! For rapid prototyping and easier deployment, Flowise is often preferred. For complex enterprise workflows, langflow might be a better fit.

Hope this helps!
